index.js

```diff
@@ -89,4 +97,25 @@ module.exports = function (processors, options) {
   }
 
   return stream
+})
+
+
+function withConfigLoader(cb) {
+  return function (plugins, options) {
+    if (typeof plugins === 'undefined') {
+      return cb(postcssLoadConfig)
```

Comment:
> As @ai said, the whole point of having the config is for it to be the same for all platforms. But if different runners injected different ctx, then it wouldn't be reusable.

Ok wait, that's a missunderstanding between us here. postcss-load-config itself exposes the `process.env.NODE_ENV && process.cwd()` => `ctx.env && ctx.cwd` by default, because there is no file I/O logic at this point. Build/Task runners like webpack, gulp, grunt, cli  have there own file I/O logic, so to make `postcss.config.js` files ubiquitous and reusable, the goal is to 'hamonize' common file I/O related `ctx` props between the runners e.g `ctx.dir, ctx.etx)` and all other runner specific stuff should be on its own prop e.g (ctx.webpack) if needed, which isn't the case for gulp I would assume.